### ISSUE TITLE
Nuke command breaks Questie until manual config wipe

### ISSUE THREAD

**Issue by SpoinkyNL** [AeroScripts/QuestieDev#200]
Hello, running /questie NUKE on Questie v2.64 I got the following error:
`Interface\Addons\!Questie\Modules\QuestieTracker.lua:538: attempt to index field 'position' (a nil value)`

Restarting doesn't help, only thing that helped was removing the config file manually.

It's reproducable by just running the command on a fresh config

**Comment by Dyaxler** [comment]
Release a hot fix for this and updated release v2.64 zip file.

**Comment by Dyaxler** [comment]
Issue resolved?

**Comment by SpoinkyNL** [comment]
Partly,

Now Questie still works after running _NUKE_, but the QuestTracker has dissapeared.
Running _cleartracker_ didn't restore it, nor did relogging

**Comment by Dyaxler** [comment]
The QuestTracker only appears when you're tracking quests...

**Comment by SpoinkyNL** [comment]
Well not anymore ;)

**Comment by Dyaxler** [comment]
Hmmm... try this and in this order.

1. Mark ALL quests as tracked - make sure you have some visual indication. EQL3 provides a check mark where as the default UI doesn't unless you have at least one quest tracked. There is a red dot when you have at least one item tracking. Green otherwise. Might be the other way around. I don't recall... :(

2. Reload UI

3. type /questie clearconfig

4. Reload UI

Still not working? Look in the layout-cache.txt file in your characters WTF folder. Look for anything that indicates Tracker or QuestTracker. Might have been another addon that added custom screen cords in there.

**Comment by Dyaxler** [comment]
Well I did manage to repro your issue. The EQL3 tracker was in it's place after doing a NUKE. I forgot that NUKE wipes EVERYTHING out and it should have detected that you have a blank config and re-inserted the defaults but it doesn't. I fixed that and it'll be in the next release.

In the mean time, I resolved it by doing a NUKE followed by a clearconfig and my quest tracker came back. The clearconfig re-inserts the default table flags where NUKE wasn't.
